### ISSUE TITLE
Nesting of headers should be recommended rule

### ISSUE THREAD

**Issue by jongund** [opena11y/oaa-rules#101]
Heading 5 and Heading 8 rules are related to proper nesting to headings are currently required.   The WCAG 2.0 working group feels this is more a of a recommended rule, so should we change these rules to recommended.

**Comment by A11YAngel** [comment]
I think they should be recommended.

**Comment by jongund** [comment]
Change the rules to be recommended.

**Comment by jongund** [comment]
Updated in OAA Library 1.1 to recommended rules
